ovc5/software/camera_device_driver/src/vdma_driver.cpp

```diff
@@ -1,42 +1,47 @@
-#include <stdlib.h>
 #include <fcntl.h>
+#include <ovc5_driver/vdma_driver.h>
+#include <stdlib.h>
+#include <string.h>
 #include <sys/mman.h>
 #include <unistd.h>
-#include <string.h>
+
+#include <cmath>
 #include <fstream>
 #include <iostream>
-#include <cmath>
-
-#include <ovc5_driver/vdma_driver.h>
 
-VDMADriver::VDMADriver(int uio_num, int cam_id) : uio(UIODriver(uio_num, UIO_SIZE))
+VDMADriver::VDMADriver(int uio_num, int cam_id)
+    : uio(UIODriver(uio_num, UIO_SIZE))
 {
   // Start by resetting the DMA
   uio.writeRegister(VDMACR, uio.readRegister(VDMACR) | (1 << 2));
-  while(uio.readRegister(VDMACR) & (1 << 2));
+  while (uio.readRegister(VDMACR) & (1 << 2))
+    ;
```

Comment:
Wow this is a strange formatting, I guess it's to highlight that it is an empty `while` loop? :thinking:

Reply:
I believe this can be changed back using:
```
AllowShortLoopsOnASingleLine (bool)
    If true, while (true) continue; can be put on a single line.
```